source/dub/dub.d

```diff
@@ -1131,6 +1133,33 @@ class Dub {
 		else return vers[$-1];
 	}
 
+	/** Returns the minimal version of a package known to compile without
+		warnings.
+
+		Contains a very limited knowledge base, only few packages and versions
+		are covered. Defaults to "0.0.0".
+
+		Params:
+			package_name: The name of the package in question.
+			frontendVersion: The version of the frontend that the package is to
+				be built with.
+	*/
+	static string getMinimalVersion(string package_name, int frontendVersion)
```

Comment:
this is a very generic name for a very specific feature, please consider making this name more concrete. (could confuse DUB as a library API users)